api/main.py

```diff
@@ -138,6 +139,32 @@ async def post_user(
     return obj
 
 
+@app.post('/user_group/{group_name}', response_model=UserGroup)
+async def post_user_group(
+        group_name: str,
+        current_user: User = Security(get_user, scopes=["admin"])):
+    """Create new user group"""
+    try:
+        obj = await db.create(UserGroup(user_group=group_name))
+        operation = 'created'
```

Comment:
I don't think we need this variable, it's only used once when calling `publish_cloudevent()`.